### DISCUSSION TITLE
morallicensing2

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#29] [Announcements]
# morallicensing2

Exploring the concept of Moral Licensing and its implications in daily life and larger societal contexts.

https://nonzerosum.games/morallicensing2.html

**Comment by andrewtaneglen** [comment]
Prison is interesting to look at through the lens of negative moral licensing. The prisoner's wrongdoing gives us moral license to severely impact their life in a way that would be considered grievously criminal if done to an innocent person. At a surface level, it seems primarily punitive/retributive, rather than restorative/rehabilitating. If there is one current civil rights thing we will look back on in 100 years and wonder about it'll probably be locking people away and then letting them back out, and thinking this is solving a problem.

**Reply by jamesstephenbrown** [reply]
Good point, I think there's significant retributive motivation with the current prison system, which would explain the resistance to changing to more rehabilitative systems even given the success of such programs abroad.

It's interesting to think of prison in terms of moral licensing if the retributive function was not performed, one might ask if victims or society at large might find an alternative outlet that might be equally negative (appealing to bothsidesism here). But I don't think this possibility is at all probable, one because these less retributive systems are used, and there seems to be no evidence of alternative negative outlets, and also because, given the idea that having clear core principles or habits counteracts moral licensing, a society that believes that rehab is the right way to treat criminals might collectively override retributive instincts.

**Comment by Nikko-A** [comment]
Hi James, once again I'd like to thank you for sharing such an interesting topic!

I think that it's interesting that in social media we are often fed negative articles/posts that feeds into the availability heuristic you mentioned you earlier. I think that a large part of why negative moral licensing has spread across the globe is due, in large part, to the onslaught of negativity that social media platforms push unto us. Platform holders, such as Meta and Twitter (currently X), do this intentionally ([See this post](https://www.dailymail.co.uk/news/article-10132759/Facebooks-algorithm-promoted-toxic-hateful-content.html) and [This post](https://www.theverge.com/2021/10/22/22740703/twitter-algorithm-right-wing-amplification-study)). Given this push of negative messaging, it further develops this sense of negative moral licensing within the minds of people.

Along with that, I'd also like to posit other systems we have that distinctly limit our ability to humanize others, one being the character limit of Twitter. Yes, we're going back to the hell-site, unfortunately. I think Twitter's micro-blogging system, limiting the majority of people's posts to 280 characters, led to the loss of nuance and humanization in online arguments. In personal arguments, you get the sense of other people’s reactions through other means, such as body language and tone. However, in online contexts we are limited to our words, unless we use voice or video but that’s incredibly rare. When you leave that little room to debate your arguments, it becomes less important to actively acknowledge the other person’s humanity and instead we dive into a more direct way of addressing arguments. Often in these cases, I notice that the argument sounds a lot more blunt and often rude, which perpetuates the image of rudeness and thus give a negative loop towards the people in the argument that feeds into negative licensing. Along with the “ratio” phenomenon and desire for viral approval (by means of having a lot of likes in posts), we just see that these platforms further deepen the negative-sum game. “I ratio’d someone, I already got my victory” becomes an actual takeaway in these scenarios.

It's unfortunate but a lot of the negative loop we're sucked into is a hell of our own creation, aided by systems that tricks us into getting pulled deeper.

**Reply by jamesstephenbrown** [reply]
I regretfully agree :) very perceptive insights there, I definitely have seen the argumentation of some people I had previously admired become retarded through the word-limit bottleneck of Twitter. Thanks for introducing me to the *verb* "ratio", I hadn't heard that one before, I'm involved in lots of discussion groups where the aim is to have long discussions, but I can see that the ratio of likes to comments could indicate unpopular ideas. The articles were also enlightening, as I had previously thought the preference for negativity in the algorithm was just an accidental byproduct of engagement maximisation, rather than a purposeful one (with 5 points awarded to angry emojis over likes).

I have tended to prefer online arguments because it helps me order my thinking and hopefully put forward more coherent points. But even with this intention it is easy to fall prey to relishing in a well-crafted barb or come back, and forgetting that the person I'm discussing with is a person who holds values in the same way I do. Having done research for this post and discovered the intractable nature of our need to perceive ourselves as good, it really has changed my understanding of what can be achieved in a discussion, and that hoping to convince someone else that they are bad is simply futile, which seems obvious once it's put that way. So, the question becomes how do I make my point and maintain the other person's sense of self-worth, which we generally do instinctively IRL.

The silver-lining is that when you do do this, people are surprised (because it is so rare) and often try to meet you half way, escaping the negative loop.

**Reply by Nikko-A** [reply]
A bit to clarify, the verb "to ratio" is not when it comes to the ratio of likes to comments, but instead about the number of likes one may get as opposed to the parent comment they are replying to. It is an idea that regardless of correctness, so long as that person is able to "ratio" the other person they are immediately correct in the argument. That's why arguments in these platforms become quippy and chase for viral sensation, rather than always choosing to add more information.

**Reply by jamesstephenbrown** [reply]
Aha, right, I see - I've definitely felt vindicated by getting more likes on my comment than on the parent comment before, an instinct I'm endeavouring to repress.
